v3/src/components/map/components/pin-controls.scss

```diff
@@ -4,7 +4,7 @@
   left: 10px;
   position: absolute;
   top: 90px;
-  z-index: 11;
+  z-index: 13;
```

Comment:
This won't have the effect you're expecting because it's within the stacking context set by the `.map-pin-layer`, which is behind the point layer.